microservices/payment/src/services/payment-gateway/stripe.ts

```diff
@@ -328,14 +352,54 @@ class Stripe extends Abstract {
       case 'payment_intent.succeeded':
         void this.handlePaymentIntent(event);
         break;
+
+      case 'charge.refund.updated':
+      case 'charge.refunded':
+        void this.handleChargeRefund(event);
+        break;
     }
   }
 
   /**
-   * Handles payment intent processing
+   * Handles refund statuses
+   */
+  public async handleChargeRefund(event: StripeSdk.Event): Promise<void> {
+    /* eslint-disable camelcase */
+    const { status, payment_intent } = event.data.object as StripeSdk.Refund;
```

Comment:
```suggestion
    const { status, payment_intent: paymentIntent } = event.data.object as StripeSdk.Refund;
```

instead disable eslint, try this hack